common/test/acceptance/pages/studio/index.py

```diff
@@ -290,6 +290,27 @@ def language_selector(self):
         )
         return self.q(css='#settings-language-value')
 
+    @property
+    def course_creation_error_message(self):
+        """
+        Returns the error text length
```

Comment:
Ideally the docstring should also be updated to more accurately reflect what this method does.  Otherwise 👍